tests/hazmat/primitives/test_serialization.py

```diff
@@ -1293,6 +1293,20 @@ def test_load_ssh_public_key(self, backend):
             b"N\x06G\xecV\xbc\x19\xaf\xc6<k\x07[\xc6"
         )
 
+    def test_public_bytes_openssh(self, backend):
+        ssh_key = (
+            b"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIG2fgpmpYO61qeAxGd0wgRaN/E4"
+            b"GR+xWvBmvxjxrB1vG user@chiron.local"
```

Comment:
Drop the comment from this input and then is this variable in the assertion at the end